app/assets/javascripts/discourse/tests/acceptance/user-menu-test.js

```diff
@@ -102,29 +104,29 @@ acceptance("User menu", function (needs) {
 
     assert.strictEqual(
       customTab1.dataset.tabNumber,
-      "5",
+      "6",
       "custom tab has the right tab number"
     );
 
     assert.strictEqual(
       customTab2.dataset.tabNumber,
-      "6",
+      "7",
       "custom tab has the right tab number"
     );
 
     const reviewQueueTab = query("#user-menu-button-review-queue");
 
     assert.strictEqual(
       reviewQueueTab.dataset.tabNumber,
-      "7",
+      "8",
       "review queue tab comes after the custom tabs"
     );
 
     const tabs = [...queryAll(".tabs-list .btn")]; // top and bottom tabs
 
     assert.deepEqual(
       tabs.map((t) => t.dataset.tabNumber),
-      ["0", "1", "2", "3", "4", "5", "6", "7", "8"],
+      ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"],
```

Comment:
Thanks for the fix, would it be worth it to create some sort of map what each number means?

Reply:
Is it better now?